src/tap.h

```diff
@@ -93,8 +122,16 @@ class TestResult {
 
   std::string toString() const {
     std::stringstream ss;
-    ss << this->status << " " << this->number << " " << this->name << " "
-       << this->getComment();
+    ss << this->status << " " << this->number << " " << this->name;
+#ifdef GTEST_TAP_13_DIAGNOSTIC
+    std::string comment_text = this->getComment();
+    if (!comment_text.empty()) {
+      ss << std::endl
+       << "# Diagnostic" << std::endl
+       << "  ---" << std::endl
+       << "  " << replace_all_copy(this->getComment(), "\n", "\n  ");
+    }
+#endif
```

Comment:
Considering that initially, [a comment was never set](https://github.com/kinow/gtest-tap-listener/blob/master/src/tap.h#L141), I think the default behavior is still the same. Have I missed something?